### PR TITLE
Added bregman-learning and torch-simplify

### PR DESCRIPTION
**Pull request by TJHeeringa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by TJHeeringa** [comment]
@conda-forge/help-python, all checks have failed and I mostly don't understand why. Can you help me?

conda-forge-linter: This one seems to be just a mistake in the linter. The repo has a meta.yml instead of a recipe.yml in it.

linux+osx: I have listed the conda-forge package igraph as dependency. In the log, it shows that it can find it
image
However, directly after it throws the error
image
Why can't it find the module? What should I change in the recipe?

windows: The build cannot find a suitable pytorch and torchvision version for the bregman-learning recipe. I have the package also on PyPI and there it works for the mentioned versions. Of course, PyPI is not Conda, but all PyTorch versions are pip and conda installable. Any pointers for what I could do?

File: recipes/bregman-learning/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - setuptools >=30.3.0
+    - wheel
+    - pip
+  run:
+    - python
+    - pytorch >=1.12.0
+    - numpy ==1.26  # [win]
+    - numpy >=1.22  # [not win]
+    - igraph
```
Comment (synapticarbors) [inline recipes/bregman-learning/meta.yaml:28]: I believe this should be `python-igraph` instead of `igraph`

**Comment by synapticarbors** [comment]
Typically we just ignore the windows failure when pytorch is present. It works otherwise and users can get pytorch from the official pytorch channel.

File: recipes/bregman-learning/meta.yaml (hunk 1 above)
Comment (TJHeeringa) [reply recipes/bregman-learning/meta.yaml]: That was it.😄

**Comment by TJHeeringa** [comment]
@conda-forge/python, ready for review!

**Pull request merged**